skunkworks_crow/src/main/java/org/odk/share/activities/SendActivity.java

```diff
@@ -106,11 +102,10 @@ protected void onCreate(Bundle savedInstanceState) {
             formIds = getIntent().getLongArrayExtra(FORM_IDS);
         }
 
-
         port = SocketUtils.getPort();
 
         if (port == -1) {
```

Comment:
`-1` can be replaced by a static variable with meaningful name, like `NO_AVAILABLE_PORT_CODE`?